projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts

```diff
@@ -185,7 +201,29 @@ async function generateForKeys(
 			if ( updateResult === false ) {
 				return;
 			}
+
+			// Track individual Critical CSS generation success.
+			const trackerAttributes: TrackerAttributes = {
+				provider_key: providerKey,
+				status: 'success',
+				size: css.length,
+				generation_time: Date.now() - providerStartTime,
+			};
+			jpTracksAJAX.record_ajax_event( 'critical_css_success', 'click', trackerAttributes );
```

Comment:
Is this a `click` event? I mean, I guess it happens well after a user clicks - but is there a more descriptive type we should use?

Reply:
Current implementation of jetpack-tracking tracks the event-props only if it is a `click` event type. I think this needs to be updated to add more meaningful event types though!
Reference: https://github.com/Automattic/jetpack/blob/master/projects/packages/tracking/src/class-tracking.php#L77